### PR TITLE
Closes AgileVentures/MetPlus_tracker#390

### PR DESCRIPTION
**Pull request by arun1595**

### CODE REVIEW

File: app/controllers/jobs_controller.rb [hunk 1]
```diff
@@ -259,6 +292,14 @@ def apply_for(job_seeker, &controller_response)
     redirect_to(job_path(@job)) && return
   end
 
+  # Sorts the match_results hash based on the `score` key, in descending order,
+  # placing nil scores at the end
+  def sort_by_score(match_results)
```
Comment (joaopapereira) [inline app/controllers/jobs_controller.rb]: Shouldn't this function be on the model? Is there any case where we would like to order by a different thing?

File: app/policies/job_policy.rb [hunk 2]
```diff
@@ -48,9 +48,17 @@ def revoke?
...
+    user && User.is_job_developer?(user)
+  end
+
   private
 
   def correct_company_person?
     user.is_a?(CompanyPerson) && (record.company == user.company)
   end
+
+  def match_jd_job_seekers?
```
Comment (joaopapereira) [inline app/policies/job_policy.rb]: Duplicated method

**Comment by joaopapereira** [comment]
Looks good, please review @patmbolger comment and try to clean Hound suggestions. We will need to rebase the PR as well

File: features/match_job_and_resumes.feature [hunk 3]
```diff
@@ -94,8 +94,8 @@ Feature: Match a Job to a Job Seeker's Résumé
     Then I should see "Signed in successfully"
     Then I click the "ruby developer" link
     And I wait 1 second
-    And I should see "Match Job Seekers"
-    Then I click the "Match Job Seekers" link
+    And I should see "Match all job seekers"
+    Then I click the "Match all job seekers" link
     And I wait 2 seconds
     Then I should see "Job Seeker matches for job: ruby developer"
     And I should see "Seeker, John"
```
Comment (patmbolger) [inline features/match_job_and_resumes.feature]: In the scenario beginning on line 105, there is this step:
```
Then I click the "Match Job Seekers" link
```
The link has changed to ```Match all job seekers```.

screen shot 2017-01-08 at 6 31 48 am

Comment (arun1595) [inline features/match_job_and_resumes.feature]: Done.

**Pull request merged**